ios/Classes/SwiftZendeskMessagingPlugin.swift

```diff
@@ -51,10 +51,29 @@ public class SwiftZendeskMessagingPlugin: NSObject, FlutterPlugin {
                 }
                 zendeskMessaging.logoutUser()
                 break
+            case "countMessages":
```

Comment:
rename

Reply:
Fixed in 20b802ed8f51d865d4df359d352885757e5c5a9c